knowledge_graph/make_network.py

```diff
@@ -49,18 +47,19 @@ def main(args):
     [give_alias(x) for x in properties]
```

Comment:
**change** the variable `properties` to "obj_properties" in the two lines above here.
AND **add** to this section the following code:
```
annot_properties = list(onto.annotation_properties())
[give_alias(x) for x in annot_properties]
```